pages/controls/Web Application Deception Technology

```diff
@@ -0,0 +1,78 @@
+---
+
+layout: col-sidebar
+title: Web Application Deception Technology
+author: Vaibhav Malik
+contributors:
+permalink: [direct link to page, removes /pages] (this is optional and requires some care)
+tags: [attack, XSS, etc]
```

Comment:
Please adjust this, look at others for guidance if you need

Reply:
Done